docs/WebAssembly.md

```diff
@@ -0,0 +1,28 @@
+# WebAssembly support in Swift
+
+WebAssembly is a platform that significantly differs from hardware platforms that Swift already supports.
+While it's a virtual machine, there are considerations to be taken into account when targeting it:
+
+* WebAssembly is still at an early stage, so many features you'd be expect from other platforms are not
+available yet, specifically:
+  1. `wasm64` variant is not specified yet, only the 32-bit `wasm32` variant is supported in WebAssembly
+  hosts such as browsers.
+  2. While a preview of multi-threading and atomics is available in some browsers, [the corresponding
+  proposal](https://github.com/WebAssembly/threads/) haven't formally reached the implementation phase yet.
```

Comment:
What about stand-alone WASM interpreters?

Reply:
Thanks, I've updated it and phrased it as "stand-alone WebAssembly hosts", as not all of those hosts are strictly interpreters, some are JIT or AOT compilers.